lib/pharos/phases/configure_host.rb

```diff
@@ -9,6 +9,9 @@ def call
         unless @host.environment.nil? || @host.environment.empty?
           logger.info { "Updating environment file ..." }
           host_configurer.update_env_file
+          host.transport.disconnect
```

Comment:
It seems that this does not work properly. @kke any ideas?

Reply:
well, one way would be to do a phase split. ConfigureHostEnvironment --> DisconnectSSH -> ConnectSSH